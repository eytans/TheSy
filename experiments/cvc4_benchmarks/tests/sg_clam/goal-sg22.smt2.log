19:21:31 [WARN] Collecting statistics
19:21:31 [WARN]  ex is: zero
19:21:31 [WARN]  ex is: (succ zero)
19:21:31 [WARN]  ex is: (succ (succ zero))
19:21:31 [WARN]  ex is: nil
19:21:31 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:21:31 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:21:31 [WARN] Running exploration without proof mode, but goals were given
19:21:31 [WARN] Proof mode: true
19:21:31 [WARN]  ex is: zero
19:21:31 [WARN]  ex is: (succ zero)
19:21:31 [WARN]  ex is: (succ (succ zero))
19:21:31 [WARN]  ex is: nil
19:21:31 [WARN]  ex is: (cons autovar_Nat_0 nil)
19:21:31 [WARN]  ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:21:31 [WARN] Running exploration without proof mode, but goals were given
19:21:31 [WARN] Proof mode: true
19:21:31 [WARN] proved: (append ?ts_ph_Lst_0 nil) => ?ts_ph_Lst_0
19:21:31 [WARN] proved: (leq zero ?ts_ph_Nat_0) => true
19:21:33 [WARN] generalized case_split proof as: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) -- (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:21:33 [WARN] proved: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) => (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:21:33 [WARN] proved: (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2)) => (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2)
19:21:33 [WARN] proved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
