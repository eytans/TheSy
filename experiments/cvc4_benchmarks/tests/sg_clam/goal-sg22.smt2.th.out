19:21:31 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(even zero) => true
true => (even zero)
(even (succ ?n)) => (not (even ?n))
(not (even ?n)) => (even (succ ?n))
(append nil ?x) => ?x
(append (cons ?x2 ?y) ?z) => (cons ?x2 (append ?y ?z))
(cons ?x2 (append ?y ?z)) => (append (cons ?x2 ?y) ?z)
(len nil) => zero
zero => (len nil)
(len (cons ?x3 ?y)) => (succ (len ?y))
(even (len (append ?w (cons ?x4 (cons ?y5 ?z))))) => (even (len (append ?w ?z)))
19:21:31 [0m[33m[WARN] [0m ex is: zero
19:21:31 [0m[33m[WARN] [0m ex is: (succ zero)
19:21:31 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:21:31 [0m[33m[WARN] [0m ex is: nil
19:21:31 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:21:31 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:21:31 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:21:31 [0m[33m[WARN] [0mProof mode: true
19:21:31 [0m[33m[WARN] [0m ex is: zero
19:21:31 [0m[33m[WARN] [0m ex is: (succ zero)
19:21:31 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:21:31 [0m[33m[WARN] [0m ex is: nil
19:21:31 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_0 nil)
19:21:31 [0m[33m[WARN] [0m ex is: (cons autovar_Nat_1 (cons autovar_Nat_0 nil))
19:21:31 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:21:31 [0m[33m[WARN] [0mProof mode: true
19:21:31 [0m[33m[WARN] [0mproved: (append ?ts_ph_Lst_0 nil) => ?ts_ph_Lst_0
19:21:31 [0m[33m[WARN] [0mproved: (leq zero ?ts_ph_Nat_0) => true
19:21:33 [0m[33m[WARN] [0mgeneralized case_split proof as: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) -- (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:21:33 [0m[33m[WARN] [0mproved: (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2) => (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2))
19:21:33 [0m[33m[WARN] [0mproved: (append ?ts_ph_Lst_0 (append ?ts_ph_Lst_1 ?ts_ph_Lst_2)) => (append (append ?ts_ph_Lst_0 ?ts_ph_Lst_1) ?ts_ph_Lst_2)
19:21:33 [0m[33m[WARN] [0mproved: (leq (succ zero) (succ ?ts_ph_Nat_0)) => true
done in 1848
